.github/workflows/prd_deploy.yaml

```diff
@@ -0,0 +1,27 @@
+name: PRD Build
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.PRD_AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.PRD_AWS_SECRET_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run export
+
+      - name: Deploy
+        uses: reggionick/s3-deploy@v2
+        with:
+          folder: out
+          bucket: ${{ secrets.PRD_S3_BUCKET }}
+          bucket-region: ${{ secrets.PRD_S3_BUCKET_REGION }}
+          dist-id: ${{ secrets.PRD_CLOUDFRONT_DISTRIBUTION_ID }}
```

Comment:
Do we need the secrets both here and in the envvars?

Reply:
@jerith I believe we do. This workflow is pretty much the same as the example in the readme https://github.com/Reggionick/s3-deploy
The action we're using doesn't accept arguments for the access keys https://github.com/Reggionick/s3-deploy/blob/master/index.js

Reply:
Whoops, I missed that it was different secrets in both places. 🤦